### PR TITLE
Products

### LINKED ISSUE #49: Backend User Story 6 - create relationship between user and products
belongs to user #35 
Currently, there is no relationship between user and products
We will have to build that relationship in the model
Update migrations
A userProducts query method needs to be created

### PR DESCRIPTION
**Pull request by kathytbui**
#### What's this PR do?
Thir pr closes #49 

#### Where should the reviewer start?
NEW MIGRATIONS AND SEEDS HAVE BEEN CREATED!! You will have to run `npm db:reset` and `npm test:reset`
This will drop your current db and recreate, remigrate, and reseed your current database.

#### How should this be manually tested?
You can go to graphiql-online.com to make a userProducts query

#### Any background context you want to provide?
A user now has userProduct which is a joins table that connects users to products

#### What are the relevant tickets?
#49 #35 

#### Questions:

### CODE REVIEW

**Comment by kip-west** [comment]
Thanks so much for getting this setup! I'm excited to get this hooked up to our FE.

**Review by kip-west — APPROVED** [review]

File: code/api/src/modules/userProduct/model.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+'use strict';
```
Comment (dtsehl) [inline code/api/src/modules/userProduct/model.js:1]: This is interesting that this auto-generated file with `sequelize` codes the file differently than the original files in the app.

**Review by dtsehl — APPROVED** [review]

**Review by ajtran303 — APPROVED** [review]

**Comment by ajtran303** [comment]
Looking good! Nice work figuring out how to write the associations

**Comment by leighlars** [comment]
Thank you for walking me through this. looks great

**Pull request merged**